README.md

```diff
@@ -17,22 +21,19 @@ Currently is is being curated by...
 > :zap: &nbsp; Vishnu Attur | @avishnu <BR>
 > :sunglasses: &nbsp; Tiago Castor | @tiagolobocastro <BR>
 
-<BR>
-<BR>
-
-> [!IMPORTANT]
-> We are a community of innovative Data Storage engineers that contribute to this project. We are greatful for the Major sponsorship that is provided by [DataCore](https://datacore.com), who donates a large team of dedicated Dev/QA/Engineers, Product Mgmt, Marketing, Business and Operational leadership as well as significant engineering infrastructure rescources and financial funding. Without DataCore, Inc... OpenEBS would not be the successful product that is it today. OpenEBS is teh result of 100's of amazing individuals, contributors and storage engineers who provide brainstorming ideas, feedback, code reviews and high-quality code to the project. - All who are passionate about storage and Data are welcome here.
->
-
-<BR>
-<BR>
+## Who we are
+We are a community of innovative Data Storage engineers that contribute to this project. We are greatful for the major sponsorship provided by [DataCore](https://datacore.com), who donate a large team of dedicated Dev/QA/Engineers, Product Mgmt, Marketing, Business and Operational leadership as well as significant engineering infrastructure rescources and financial funding. Without DataCore, Inc... OpenEBS would not be the successful product that is it today. OpenEBS is the result of 100's of amazing individuals, contributors and storage engineers who provide brainstorming ideas, feedback, code reviews and high-quality code to the project. Anyone who is passionate about storage and data is welcome here.
+[![OpenEBS Welcome Banner](/images/openebs_community_banner_retro_gamer.png)](https://www.openebs.io/)
 
 > [!NOTE]
 > CNCF Project status : ARCHIVE <BR>
-> In Jan 2024, under the guidnace of CNCF TOC, our project moved from Sandbox level to Archvie level so that we could impliemnt an extensive restructing initative. We are actively archiving out old projects, repos and dependancies to a CNCF owned repo called: OpeneEBS-Archive (https://github.com/openebs-archive).<BR>
-> The product team is unifying 4 main repos (Mayastor, LocalPV-LVM, LocalPV-ZFS, LocalPV-Hostpath) and their support technbology stack repos into one core OpnenEBS code-base. See our main repo for more details.
+> In Jan 2024, under the guidance of CNCF TOC, our project moved from Sandbox level to Archive level, enabling us to implement an extensive restructing initative. We are actively working to archive old projects, repos and dependancies to a CNCF owned repo called: OpeneEBS-Archive (https://github.com/openebs-archive), plus update our governance and operating procedures.<BR>
```

Comment:
archive legacy sub projects, repositories and dependencies to a CNCF-owned GitHub organization called ...